VocaDbWeb/Scripts/Shared/WebLinkMatcher.ts

```diff
@@ -3,9 +3,9 @@ import _ from 'lodash';
 
 export default class WebLinkMatcher {
 	public static matchers: WebLinkMatcher[] = [
-		{ url: '5sing.kugou.com/', desc: '5SING', cat: WebLinkCategory.Official },
-		{ url: 'about.me/', desc: 'about.me', cat: WebLinkCategory.Official },
-		{ url: '.akbh.jp/', desc: 'Akiba Hobby', cat: WebLinkCategory.Commercial },

```

Comment:
Hmm, I'm not sure how you did this, but this is not what I meant. After installing and [setting up](https://github.com/VocaDB/vocadb/wiki/VocaDB-development-environment-(Windows)#visual-studio-code) the [prettier code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) in [vscode](https://code.visualstudio.com/), you should be able to automatically format source code based on the [pre defined configuration](https://github.com/VocaDB/vocadb/blob/7eca9ac675dede7dfe6385deac128d72e3260b29/VocaDbWeb/.prettierrc), for example, by pressing ctrl+s.

Reply:
I tried to run Prettier in JB Rider at first, but it doesn't seem to be fully compatible with it